src/phongo_compat.h

```diff
@@ -303,4 +303,42 @@ zend_bool zend_array_is_list(zend_array* array);
 typedef ZEND_RESULT_CODE zend_result;
 #endif
 
+/* get_gc iterator handler was added in PHP 8.0 */
+#if PHP_VERSION_ID >= 80000
+#define PHONGO_ITERATOR_FUNCS(dtor, valid, get_current_data, get_current_key, move_forward, rewind, invalidate_current, get_gc) \
+	{                                                                                                                           \
+		(dtor),                                                                                                                 \
+			(valid),                                                                                                            \
+			(get_current_data),                                                                                                 \
+			(get_current_key),                                                                                                  \
+			(move_forward),                                                                                                     \
+			(rewind),                                                                                                           \
+			(invalidate_current),                                                                                               \
+			(get_gc),                                                                                                           \
+	}
+#else
```

Comment:
```suggestion
#else /* PHP_VERSION_ID < 80000 */
```

This is used in some other longer `#if` blocks elsewhere in the file.

Note that we don't have prior art for this with `#if(n)def`, so I didn't suggest it below.